lang/syn/src/parser/accounts/mod.rs

```diff
@@ -120,6 +120,25 @@ fn constraints_cross_checks(fields: &[AccountField]) -> ParseResult<()> {
                     ));
                 }
             }
+            match kind {
+                // this doesn't catch cases like
```

Comment:
Let's format the comment a bit. 80 chars per line + proper punctuation + capitalization. :)

Reply:
will do. should the 80 chars include the whitespace at the beginning of the line?

Reply:
will do